packages/realm/src/ProgressRealmPromise.ts

```diff
@@ -57,6 +58,19 @@ function determineBehavior(config: Configuration, realmExists: boolean): OpenBeh
 }
 
 export class ProgressRealmPromise implements Promise<Realm> {
+  /** @internal */
+  private static instances = new Set<binding.WeakRef<ProgressRealmPromise>>();
+  /**
+   * Cancels all unresolved `ProgressRealmPromise` instances.
+   * @internal
+   */
+  public static cancelAll() {
```

Comment:
It seems weird to have a method in the middle of your instance variable declarations. Move this down?

Reply:
I consider it "normal" to have all static members and methods before any instance methods and properties.

Reply:
Fair enough. Could you at least separate the sections with a blank line then?